server.js

```diff
@@ -0,0 +1,72 @@
+var express = require("express");
+var app = express();
+var bodyparser = require("body-parser");
+var port = process.env.PORT || 3000; //listen to requests
+// on port 3000 - that is what is going on here
+// you can set port to something else but 3000 is standard
+
+var piglatinify = require("./lib/piglatinify.js");
+var basicYou = require("./lib/basicinfo.js");
+var quotes = require("./lib/quotes.js");
+var jokes = require("./lib/jokes.js");
+//var favorite = require("./lib/favorite.js");
+//var maybe = require("./lib/quotes.js");
+app.use(bodyparser.json());
+app.use(bodyparser.urlencoded({ extended: true }));
+
+//app.use(express.static(__dirname + "/app/"));
+
+/*var information	 = ["Do Better", "Try Harder", "You Got It", "Dont Stop Yet", "Awesome"];
+*/
+app.use(express.static(__dirname + "/app/"));
+
+var favorite = [
+{ favoritestadium: "Wrigley Field",
+reasoning: "The ivy is gorgeous and nothing beats the bleachers in the middle of summer!" },
+{ favoritestadium: "Fenway Park",
+reasoning: "The green monster" },
+{ favoritestadium: "Dodger Stadium",
+reasoning: "Los Angeles can not be beat. City baseball" }
+];
+
+app.get("/", function(req, res) {
+	res.sendFile("index.html");
+});
+
+app.get("/quote", function(req, res) {
+	var randomIndex = Math.floor(Math.random() * quotes.length);
```

Comment:
right now, your module exports from the quote file returns an array, so this works. The next thing to do would be to change your module export into a function, move this line of code into the function, and have the function return the random quote when called. The setup would be similar to piglatinify.js.